src/web/components/Pillars.tsx

```diff
@@ -215,31 +224,34 @@ const isNotLastPillar = (i: number, noOfPillars: number): boolean =>
 
 export const Pillars: React.FC<{
     display: Display;
-    mainMenuOpen: boolean;
+    menuCheckboxId?: string;
```

Comment:
Might be better to have a more descriptive prop here like, `showFoobar: boolean` rather than use the id (which is fixed) as a flag

Reply:
we need to know the `menuCheckboxId` as it is what is used in the CSS to get the ID of the checkbox. The boolean was removed as we are supporting NoJS and we do not want components to have internal states

Reply:
But isn't `menuCheckboxId ` a static string. Why does it need to be a prop if it never changes?